js/reducers/dashboard-reducer.js

```diff
@@ -2,14 +2,16 @@ import { Map } from 'immutable'
 import {
   SET_ACTIVITY_EXPANDED, SET_STUDENT_EXPANDED,
   SET_STUDENTS_EXPANDED, SET_STUDENT_SORT,
-  SORT_BY_NAME, SET_QUESTION_EXPANDED
+  SORT_BY_NAME, SET_QUESTION_EXPANDED,
+  SELECT_QUESTION
 } from '../actions/dashboard'
 
 const INITIAL_DASHBOARD_STATE = Map({
   sortBy: SORT_BY_NAME,
   expandedActivities: Map(),
   expandedStudents: Map(),
-  expandedQuestions: Map()
+  expandedQuestions: Map(),
+  selectedQuestion: Map()
```

Comment:
Is this Map() a map of selected questions or just a question state?
If it a single question, I'd perhaps expect `null` here or ID instead of a copy of data. Copying can have some side effects if the data is updated and they will be out of sync. And that can actually happen, as the dashboard is meant to be used in "live" mode, so the new answers can show up.